### PR TITLE
Fix VHostScan installation issues

### PR DESCRIPTION
**Pull request by ibitebyt3s**
People seem to be struggling to install VHostScan as in issues [#122](https://github.com/codingo/VHostScan/issues/122) and [#123](https://github.com/codingo/VHostScan/issues/123) and I found that by removing the version numbers in `requirements.txt` The install and test scripts would run flawlessly. 

I also updated `test-requirements.txt` to follow the same "version numbering" in both `*.txt` files. 

Lastly, I removed `pytest` command example from the `README.md` because it uses python2 which is now deprecated. The problem with it is that it was trying to import the `ipaddress` module and since `pip2` is not available, it would be problem to install it and keep building on top of it. 

In conclusion the following commands should run without issues:

```bash
sudo python3 setup.py install
sudo python3 setup.py test
```

### CODE REVIEW

**Comment by aps-kar** [comment]
I had the same issues installing but your fork seems to work, thank you.

**Comment by ibitebyt3s** [comment]
You might be calling the script with:
```
python3 VHostScan/VHostScan.py                                  
```
instead, call it as any binary in the `$PATH variable`. Just type `VHostScan` and it should be up and running.

---
```
user@hostname:~/VHostScan$ python3 VHostScan/VHostScan.py           
Traceback (most recent call last):
  File "VHostScan/VHostScan.py", line 8, in <module>
    from .lib.core.virtual_host_scanner import virtual_host_scanner
ImportError: attempted relative import with no known parent package
```

```
user@hostname:~/VHostScan$ VHostScan     
+-+-+-+-+-+-+-+-+-+  v. 1.21
|V|H|o|s|t|S|c|a|n|  Developed by @codingo_ & @__timk
+-+-+-+-+-+-+-+-+-+  https://github.com/codingo/VHostScan

usage: VHostScan [-h] -t TARGET_HOSTS [-w WORDLISTS] [-b BASE_HOST] [-p PORT] [--prefix PREFIX] [--suffix SUFFIX] [-r REAL_PORT] [--ignore-http-codes IGNORE_HTTP_CODES]
                 [--ignore-content-length IGNORE_CONTENT_LENGTH] [--first-hit] [--unique-depth UNIQUE_DEPTH] [--ssl] [--fuzzy-logic] [--no-lookups] [--rate-limit RATE_LIMIT] [--waf] [-v]
                 [-oN OUTPUT_NORMAL | -oJ OUTPUT_JSON | -oG OUTPUT_GREPABLE] [--random-agent | --user-agent USER_AGENT]
VHostScan: error: the following arguments are required: -t
```

**Comment by theguly** [comment]
latest requests (2.26) breaks the code, using requests==2.20.0 works like a charm.

for the records, i had it working doing:
```
apt-get install subversion python3-venv
ln -s /usr/include/locale.h /usr/include/xlocale.h
fix requirements.txt

python3 -m venv .venv
. .venv/bin/activate
pip install wheel
pip install numpy
pip install requests==2.20.0
python3 setup.py install
```
